HW_6/HW_5a.py

```diff
@@ -0,0 +1,144 @@
+
+import datetime
+import random
+import datefinder
+import HW_3_updated as normalization
+import sys, os
+
+sys.path.append('C:\\Users\\Kateryna_Kulmatytska\\PycharmProjects\\Python_HW\\HW_6')
+
+
+class Article:
+    def __init__(self, header, text):
+        self.header = header
+        self.text = text
+
+    def print_to_file(self, formatted_text, file_name):
+        with open(file_name, "a") as file:
+            file.write(formatted_text)
+
+
+class News(Article):
+    def __init__(self, header, text, location):
+        Article.__init__(self, header=header, text=text)
+        self.location = location
+
+    def format_text(self, header, text, location):
+        return f"{header}--------------" \
+               f"\n{text}\n" \
+               f"{location},{datetime.datetime.today().strftime('%d/%m/%Y %H.%M')}" \
+               f"\n-------------------\n\n"
+
+
+class PrivatAdv(Article):
+    def __init__(self, header, text, expiration_date):
+        Article.__init__(self, header=header, text=text)
+        self.expiration_date = expiration_date
+
+    def format_text(self, header, text, expiration_date):
+        return f"{header}----------" \
+               f"\n{text}\n" \
+               f"Actual until: {expiration_date}, {(expiration_date - datetime.date.today()).days} days left." \
+               f"\n-----------------------\n\n"
+
+
+class Consider(Article):
+    def __init__(self, header, text, score):
+        Article.__init__(self, header=header, text=text)
+        self.score = score
+
+    def format_text(self, header, text, score):
+        return f"{header}-------------- " \
+               f"\n{text}\n" \
+               f"Probability of truth:{score}%" \
+               f"\n------------------------------\n\n"
+
+
+def write_article():
```

Comment:
This function contains 80 lines of code. The python way is to create small functions (20-30 lines) in order to have understandable code.